src/Symfony/Component/Translation/Bridge/Lokalise/LokaliseProvider.php

```diff
@@ -250,43 +250,57 @@ private function updateTranslations(array $keysByDomain, TranslatorBagInterface
             }
         }
 
-        $chunks = array_chunk($keysToUpdate, 500);
-        $responses = [];
-
-        foreach ($chunks as $chunk) {
-            $responses[] = $this->client->request('PUT', 'keys', [
-                'json' => ['keys' => $chunk],
-            ]);
-        }
+        $response = $this->client->request('PUT', 'keys', [
+            'json' => ['keys' => $keysToUpdate],
+        ]);
 
-        foreach ($responses as $response) {
-            if (200 !== $response->getStatusCode()) {
-                $this->logger->error(sprintf('Unable to create/update translations to Lokalise: "%s".', $response->getContent(false)));
-            }
+        if (200 !== $response->getStatusCode()) {
+            $this->logger->error(sprintf('Unable to create/update translations to Lokalise: "%s".', $response->getContent(false)));
         }
     }
 
-    private function getKeysIds(array $keys, string $domain): array
+    private function getKeysIds(array $keys, string $domain, int $page = 1, int $limit = 5000): array
```

Comment:
no need to add an argument for the limit, as you never use it with a different value than the default

Reply:
Thank u. I wanted to use it somehow, but now it's actually can be removed. I fixed it.